docs/30DaysOfPWA/advanced-capabilities/06.md

```diff
@@ -1,15 +1,174 @@
+
+## Privacy and resource utilization considerations
+
+Both Background Sync and Periodic Background Sync APIs are designed to run some custom code (that might send network requests) in the background, at any time, and without notifying a user. It raises at least two concerns: privacy and resource utilization. Any network request is potentially a history leaking and location tracking reason. And any code consumes CPU, memory, battery, and, if needed, network resources. To avoid these problems, both specifications have requirements and recommendations that browser vendors must follow while implementing these APIs. They include:
+
+- Having Background Sync permission granted for the origin. It has `allow` state by default but users can deny it.
+
+![Background Sync permission](_media/background-sync-permission.png)
+
+- Capping the number of background sync retries and duration of periodic syncs. It's not defined in API specification, but for the periodic sync intervals, browsers use the _Site Engagement_ index (number and duration of user interactions) for the particular origin on a particular device to calculate the exact intervals. At best it is a few hours. Hint: you can check the Site Engagement index for yourself on your current Chromium-based browser by visiting [about://site-engagement/](about://site-engagement/) url.
+- Code execution time is limited by the service worker's lifetime which is a few seconds. So abusing device hardware by heavy computation or overusing network resources is rather difficult. For downloading the large files and having their content available for your frontend app, you can use [Background Fetch API](https://docs.microsoft.com/en-us/microsoft-edge/progressive-web-apps-chromium/how-to/background-syncs#use-the-background-fetch-api-to-fetch-large-files-when-the-app-or-service-worker-isnt-running) from the same background services family. This API is designed to keep the user in full control of the long network operation by displaying a browser native UI component with the download progress and Pause/Cancel/Resume buttons.
+- Both `sync` and `periodicsync` events are only fired online. For Background Sync detecting online is a main and only purpose. While for Periodic Background Sync this limitation emphasizes the intended use case: keeping the application itself and/or its data up-to-date by periodically syncing with the server.
+
+Periodic Background Sync API has a few additional requirements:
+
+- The `periodicsync` will happen only on a known network (the one user was connected to before).
+- It only works for _installed_ web applications.
+
+Please note, that the limitations listed above are a current "snapshot" which is a mix of MAY, SHOULD, MUST statements of the respective specifications and some additional voluntary steps from the browser vendors.
```

Comment:
```suggestion
Please note, that the limitations listed above are subject to evolve in the future.
```

Trying to make it simpler, since our readers might not be super familiar with spec work and vocabulary.

Reply:
After another look, I removed this statement completely